docs/src/doc/versioning.md

```diff
@@ -1,7 +1,7 @@
 The module uses semantic versioning for its own versions but adds a suffix for the supported godot version:
 
-Full version: `0.1.0-3.2.3`
+Full version: `0.1.0-3.3.0`
```

Comment:
nit: I don't think its scalable to change this documentation every time we bump the engine version. The versions here are an example, so this specific change is unnecessary.